docs/concepts/what-is-a-codec-server.md

```diff
@@ -0,0 +1,28 @@
+---
+id: what-is-a-codec-server
+title: What is a Codec Server?
+sidebar_label: Codec Server
+description: Explanation and implementation of a remote encryption/decryption server.
+---
+
+A Codec Server is an optional feature used to encrypt and decrypt Payloads for the WebUI.
+
+![](/img/remote-codec-server-problem.svg)
+
+Each Client has a built-in [Data Converter](/docs/concepts/what-is-a-data-converter). However, the pre-built tctl and WebUI binaries won't be able to decrypt the serialized Payloads.
```

Comment:
```suggestion
Each Temporal Client has a built-in [Data Converter](/docs/concepts/what-is-a-data-converter) that serializes Payloads.
However, the pre-built tctl and WebUI binaries are unable to decrypt the serialized Payloads.
```

We are using the terms "serialize", "encode", and "encrypt" -
Let's make sure we are using them correctly.

Serializing is about structuring data
Encoding is a general term for transforming the data
Encrypting is about transforming the data using a key and algorithm

The setting in the Web UI is called "Data Encoder".
And then you can use either the tctl Plugin or the Codec Server.

Obviously the WebUI can handle the serialized data?

So, it seems like the Codec Server is specifically for any handling any other level of encoding/decoding that is happening to the data - including encryption?